docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,131 @@
+   
+   ![Project Wizard](media/install-extension7.png)
+
+## Install Tizen Baseline SDK
+
+Baseline SDK contains Tizen-specific libraries and tools such as profiler, debugger, and so on.
+
+After installing the Visual Studio Mac Extension for Tizen, you must set up the Tizen Baseline SDK. You can either:
+
+- [Install a new Tizen Baseline SDK](#install-new-tizen-baseline-sdk) if you have not already installed the SDK.
+- [Configure an existing Tizen Baseline SDK](#configure-existing-tizen-baseline-sdk) when you want to use the installed SDK.
+
+### Install new Tizen Baseline SDK
+
+1. In the Visual Studio Mac IDE menu, go to **Tools > Tizen > Tizen Package Manager**.
+2. Select **Install new Tizen SDK**.
+  
+   ![Select new installation](media/howtoinstall-installwizard1.png)
+
+3. Read the license document and click **I Agree**.
+
+   ![Agree to license details](media/howtoinstall-installwizard2.png)
+
+4. Enter the root directory path where you want to install the Tizen Baseline SDK and click **Next**.
+
+   ![Set the installation path](media/howtoinstall-installwizard3.png)
+
+   The Tizen SDK installer is downloaded and the Baseline SDK is installed automatically.
+
+   ![Installer download](media/howtoinstall-installwizard4.png)
+
+5. In the **Package Manager** window that appears, select **Main SDK**.
+
+6. Select **Tizen SDK tools** and click **install**. You can see the installation progress in the **Progress** tab.
+
+   ![Tool installation](media/howtoinstall-installwizard6.png)
+
+<a name="configure-existing-tizen-baseline-sdk"></a>
+### Configure existing Tizen Baseline SDK 
+
+You can also use Tizen Package Manager to configure the Tizen Baseline SDK path and each tool path directly:
+
+- To set up the Tizen Baseline SDK path:
+
+  1. In the Visual Studio Mac IDE menu, select **Tools > Tizen > Tizen Package Manager**.
+  2. Select **Use installed Tizen SDK**.
+
+     ![Baseline SDK Install](media/howtoinstall-installwizard7.png)
+
+  3. Enter the root directory of your existing Tizen Baseline SDK installation and click **OK**.
+
+     ![Baseline SDK Install](media/howtoinstall-installwizard8.png)
+	 
+     The Tizen Baseline SDK is installed automatically.
+
+     > [!NOTE]
+     > If the installer gives a warning about your Tizen Studio version being too low, update the Tizen Baseline SDK by using the Tizen Package Manager after setting the tool path.
+
+- To set up each tool path directly:
+
+  1. In the Visual Studio Mac IDE menu, select **Project > Solution Options > Tizen > Tools**.
+  2. Enter the root directory of your existing Tizen Studio installation in the **Tool Path(Tizen SDK)** field.
+
+     ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
+
+     The other tools paths are configured automatically.
+
+
+## Troubleshoot
+
+If you encounter any issue with the installation, verify whether the Tizen Baseline SDK is installed correctly. 

```

Comment:
Please remove article "the" before Tizen Baseline SDK:

If you encounter any issue with the installation, verify whether Tizen Baseline SDK is installed correctly.